packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Updates android example to be able to use custom use artifactregistery.
```

Comment:
Since nobody outside the Flutter team can use this, let's not put it in the CHANGELOG.

Reply:
They can if they setup a artifactrepository then setup the env variables on their local machine (I just updated the file to reflect this). I removed the google specific access parts to configuration. Also mechanically there is a new dependency and the changelog thing scolded me. 

I can remove this if you want even though there is technically a developer facing change that should not impact the.

Reply:
removed and replaced with https://github.com/flutter/flutter/wiki/Plugins-and-Packages-repository-structure#gradle-structure